luigi/scheduler.py

```diff
@@ -363,17 +363,19 @@ def prune(self, config):
         if self.last_active + config.worker_disconnect_delay < time.time():
             return True
 
-    def get_pending_tasks(self, state):
+    def get_pending_tasks(self, state, include_failed=False):
```

Comment:
Is it possible to instead implement a `get_statuses([list of statuses])`? I've always found it a bit weird that `get_pending_tasks` also returns running tasks. This could be an opportunity to fix that.